eyre/tests/test_location.rs

```diff
@@ -100,12 +100,13 @@ fn test_option_ok_or_eyre() {
 #[cfg(feature = "anyhow")]
 #[test]
 fn test_context() {
+    use eyre::ContextCompat;
```

Comment:
For consistency with the other tests, this `use` could be moved to after the hook setup, just before _using_ `ContextCompat::with_context`.

Or the existing `use`s moved to the start of each test fn as well. 

Examples of `use` right before trait usage:

- https://github.com/eyre-rs/eyre/pull/150/files#diff-1ff47dac6cf55e34ff587968c5b1f1ec6b6ae39d2668a66ecba3633163a21fc5L142-R145 
- https://github.com/eyre-rs/eyre/pull/150/files#diff-1ff47dac6cf55e34ff587968c5b1f1ec6b6ae39d2668a66ecba3633163a21fc5L157-R160